components/playback/controls/player/PopInput.vue

```diff
@@ -0,0 +1,231 @@
+<template>
+  <div>
+    <div
+      class="div__popinput-background"
+      :style="
+        'width: ' +
+        input_width_background +
+        'px;' +
+        'height: ' +
+        input_height_background +
+        'px;'
+      "
+    >
+      <span
+        v-if="title_label !== ''"
+        class="span__popinput-content-label"
+        :style="'width: ' + input_width + 'px;'"
+      >
+        {{ title_label }}
+        <!--  original mockflow ID: cmpDb072c1da7a823374cbee04cb1666edb1   -->
+      </span>
+
+      <div
+        class="div__popinput-controls-content-input-widget"
+        :class="[
+          !input_key_action
+            ? 'div__popinput-controls-content-input--invalid-widget'
+            : 'div__popinput-controls-content-input--valid-widget',
+        ]"
+        :style="
+          'width: ' + input_width + 'px;' + 'top:' + input_widget_top + 'px;'
+        "
+      >
+        <span
+          class="span__popinput-controls-content-input-txt-widget"
+          :style="'width: ' + input_width + 'px;'"
+        >
+          <b-form-input
+            id="input-widget"
+            v-model="input_value_key"
+            :spellcheck="input_check"
+            :state="input_key_action"
+            aria-describedby="input-feedback"
+            :placeholder="key_placeholder"
+            :disabled="block"
+            class="w-100 h-100 edit-id"
+            style="
+              border-radius: 0;
+              color: rgb(255, 255, 255);
+              background-color: #3f3f3f;
+              border: 0px;
+            "
+          ></b-form-input>
+        </span>
+      </div>
+      <div
+        class="div__popinput-controls-content-input-feedback"
+        :style="
+          'width: ' + input_width + 'px;' + 'top:' + input_feedback_top + 'px;'
+        "
+      >
+        {{ invalid_text }}
+      </div>
+    </div>
+  </div>
+</template>
+<script>
+import Vue from "vue";
+import uuid from "@tofandel/uuid-base62";
+import BootstrapVue from "bootstrap-vue";
+import { BButton } from "bootstrap-vue";
+import { BFormInput } from "bootstrap-vue";
+Vue.use(BootstrapVue);
+Vue.component("BFormInput", BFormInput);
+Vue.component("BButton", BButton);
+import "bootstrap/dist/css/bootstrap.min.css";
+Vue.use(uuid);
+
+export default {
+  name: "PopInput",
+  props: {
+    title_label: { type: String, default: "", required: false }, // title_text (str) (optional, defaults to empty string "")
+    key_placeholder: { type: String, default: "" }, // placeholder (str)
+    invalid_text: { type: String, default: "" }, // invalid_text (str)
+    input_check: { type: Boolean, default: true, required: false }, // spellcheck (optional bool=True)
+    user_key: { type: String, default: "", required: false }, // field_value (str) (optional, defaults to empty string "")
```

Comment:
is there a reason you want to make up a new name `user_key` for this? can we just use `value` so it has a similar and familiar syntax as the disabled used for the input DOM element